_includes/IDR_API_example_script.md

```diff
@@ -0,0 +1,64 @@
+
+## Example of using the IDR web API
+
+OMERO.web uses a default session backend authentication scheme for authentication.
+First create a HTTP session using the `requests` library:
```

Comment:
link `requests` to http://docs.python-requests.org/en/master/user/advanced/#session-objects